_monitoring-plugins/ad/api.md

```diff
@@ -368,297 +380,1546 @@ POST _plugins/_anomaly_detection/detectors/<detectorId>/_preview
+          "execution_start_time": 1633388922742,
+          "state": "FINISHED",
+          "coordinating_node": "2Z4q22BySEyzakYt_A0A2A",
+          "task_type": "HISTORICAL_HC_DETECTOR",
+          "execution_end_time": 1633389090738,
+          "started_by": "admin",
+          "init_progress": 0,
+          "is_latest": true,
+          "detector": {
+            "category_field": [
+              "error_type"
+            ],
+            "description": "test",
+            "ui_metadata": {
+              "features": {
+                "test_feature": {
+                  "aggregationBy": "sum",
+                  "aggregationOf": "value",
+                  "featureType": "simple_aggs"
+                }
+              },
+              "filters": []
+            },
+            "feature_attributes": [
+              {
+                "feature_id": "ZS5zTXwBwf_U8gjUTfIn",
+                "feature_enabled": true,
+                "feature_name": "test_feature",
+                "aggregation_query": {
+                  "test_feature": {
+                    "sum": {
+                      "field": "value"
+                    }
+                  }
+                }
+              }
+            ],
+            "schema_version": 0,
+            "time_field": "timestamp",
+            "last_update_time": 1633386974533,
+            "indices": [
+              "server_log"
+            ],
+            "window_delay": {
+              "period": {
+                "unit": "Minutes",
+                "interval": 1
+              }
+            },
+            "detection_interval": {
+              "period": {
+                "unit": "Minutes",
+                "interval": 5
+              }
+            },
+            "name": "testhc",
+            "filter_query": {
+              "match_all": {
+                "boost": 1
+              }
+            },
+            "shingle_size": 8,
+            "user": {
+              "backend_roles": [
+                "admin"
+              ],
+              "custom_attribute_names": [],
+              "roles": [
+                "own_index",
+                "all_access"
+              ],
+              "name": "admin",
+              "user_requested_tenant": "__user__"
+            },
+            "detector_type": "MULTI_ENTITY"
+          },
+          "user": {
+            "backend_roles": [
+              "admin"
+            ],
+            "custom_attribute_names": [],
+            "roles": [
+              "own_index",
+              "all_access"
+            ],
+            "name": "admin",
+            "user_requested_tenant": "__user__"
+          }
         }
       }
     ]
   }
 }
 ```
 
-In high cardinality detectors, the result contains entity information.
-
-To see an ordered set of anomaly records for an entity with an anomaly within a certain time range for a specific feature value:
+To search for the latest entity-level task for a high cardinality detector:
```

Comment:
Your example limits the tasks to a historical analysis. Also, you are searching for all latest tasks, not a single task.  How about "To search for the latest entity-level tasks for the historical analysis of a high cardinality detector:"?

Reply:
Agree with Kaituo, that description is more accurate.